routing/probability_bimodal.go

```diff
@@ -301,6 +301,28 @@ func (p *BimodalEstimator) calculateProbability(directProbability float64,
 		return directProbability
 	}
 
+	// If we have up-to-date information about the channel we want to use,
+	// i.e. the info stems from results no longer ago than the decay time,
+	// we will only use the direct probability. This is needed in order to
+	// avoid that other results of the same node will distort the calculated
```

Comment:
> other results of the same node

not sure that I understand, what do you mean here? Can the same path have different probabilities because of multiple updates from different sending attempts?